taskcluster/client.py

```diff
@@ -490,6 +490,12 @@ def _makeHttpRequest(self, method, route, payload):
                     "Failed to establish connection",
                     superExc=rerr
                 )
+                
+            # Handles Non JSON responses
+            answer = response.get_content_type()
+            typecheck = "application/json"
```

Comment:
The name is a little confusing, It's not really a check per se, but rather a value.  Maybe `contentType` is a little more clear?